server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableImpl.java

```diff
@@ -641,13 +643,18 @@ public void compact(CompactionServiceId service, CompactionJob job, RateLimiter
     }
 
     StoredTabletFile metaFile = null;
+    long startTime = System.currentTimeMillis();
+    long entriesWritten = 0;
+    boolean failed = false;
+    CompactionStats stats = new CompactionStats();
     try {
 
       TabletLogger.compacting(getExtent(), job, localCompactionCfg);
+      tablet.incrementStatusMajor();
```

Comment:
What does this method do?  If its nots quick to explain just let me know, I can look at the source.

Reply:
It increments the counter for ongoing major compactions. If you are wondering about synchronization, there is none. The stats are stored in a thrift object called `ActionStats`.